### PR TITLE
New: (Indexers) (Abnormal) Add the indexer

### PR DESCRIPTION
**Pull request by AthAshino**
Adding the indexer

### CODE REVIEW

File: definitions/v1/abnormal.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+id: Abnormal
```
Comment (Qstick) [inline definitions/v1/abnormal.yml:2]: Id to be lower

File: definitions/v1/abnormal.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+name: ABN
```
Comment (Qstick) [inline definitions/v1/abnormal.yml:3]: Make this full name

**Review by Qstick — CHANGES_REQUESTED** [review]

File: definitions/v1/abnormal.yml (hunk 1 above)
Comment (bakerboy448) [reply definitions/v1/abnormal.yml:2]: can we lint this at all?

Comment (AthAshino) [reply definitions/v1/abnormal.yml]: Done

File: definitions/v1/abnormal.yml (hunk 2 above)
Comment (AthAshino) [reply definitions/v1/abnormal.yml]: Done

**Comment by JigSawFr** [comment]
@AthAshino So you didn't make the choice to use API of tracker ?

File: definitions/v1/abnormal.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
+---
+id: abnormal
+name: Abnormal
+description: "General French Private Tracker"
+language: fr-fr
+type: private
+encoding: UTF-8
+links:
+  - https://abn.lol/ # site does not support https ERR_CONNECTION_CLOSED
```
Comment (bakerboy448) [inline definitions/v1/abnormal.yml:9]: comment of not supporting https does not match the fact that it is an https link?

Comment (AthAshino) [reply definitions/v1/abnormal.yml]: Oh, it was a bad comment, forgot to remove it

**Comment by JigSawFr** [comment]
> > @AthAshino So you didn't make the choice to use API of tracker ?
> 
> 
> 
> Hello, there is no search api for ABN, so no other option for now

API is capable to find TMDB ID, at last. So it could be fine ? Additionally it's more efficient than querying a string are results  match 100% request. If I remember Radarr can send TMDB request, don't know for Sonarr

**Comment by Qstick** [comment]
> > > @AthAshino So you didn't make the choice to use API of tracker ?
> > 
> > 
> > 
> > Hello, there is no search api for ABN, so no other option for now
> 
> API is capable to find TMDB ID, at last. So it could be fine ? Additionally it's more efficient than querying a string are results  match 100% request. If I remember Radarr can send TMDB request, don't know for Sonarr

Correct, Radarr will send them. No tmdb from Sonarr, just IMDb and tvdb requests

**Comment by AthAshino** [comment]
> > > @AthAshino So you didn't make the choice to use API of tracker ?
> > 
> > 
> > Hello, there is no search api for ABN, so no other option for now
> 
> API is capable to find TMDB ID, at last. So it could be fine ? Additionally it's more efficient than querying a string are results match 100% request. If I remember Radarr can send TMDB request, don't know for Sonarr

I think you're not talking about ABN, ABN doesn't have any search API, the API is only for upload/search specific release. At least it's what the developer told me so...

**Review by Qstick — APPROVED** [review]

**Pull request merged**